spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe "Guest Show Page" do 
+  before(:each) do 
```

Comment:
Nice job using a `before :each`, and you have really good test data to work from 👍